cmd/livez_handler_test.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"net/http"
+	"net/http/httptest"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("livezHandler", func() {
+	var (
+		recorder *httptest.ResponseRecorder
+		request  *http.Request
+	)
+
+	BeforeEach(func() {
+		recorder = httptest.NewRecorder()
+		request, _ = http.NewRequest("GET", "/livez", nil)
```

Comment:
Can you add error handling for new requests

Reply:
done